src/main/java/games/tictactoe/TicTacToeGameState.java

```diff
@@ -57,7 +60,11 @@ protected double _getHeuristicScore(int playerId) {
      */
     @Override
     public double getGameScore(int playerId) {
-        return 0;
+        if (playerResults[playerId] == Utils.GameResult.WIN)
```

Comment:
In ExplodingKittens update it's 1 for WIN and -1 for LOSE, this should be somewhat consistent. In fact, the GameResult enum has values associated, I'd suggest using those instead of arbitrary numbers different across games.

Reply:
Cool! I have taken that as licence to re-write the main sets of these values. (Having LOSE, GAME_ONGOING and DRAW all equal to 0.0 annoyed me.) Now standardised across the board!